docs/windows.md

```diff
@@ -208,7 +208,7 @@ your own OVS package with a signed OVS kernel driver, you would run:
 
 ```powershell
 curl.exe -LO https://raw.githubusercontent.com/antrea-io/antrea/main/hack/windows/Install-OVS.ps1
-.\Install-OVS.ps1 -ImportCertificate $false -Local -LocalFile <PathToOVSPackage>
+.\Install-OVS.ps1 -InstallUserspace $true -LocalFile <PathToOVSPackage>
```

Comment:
Since `InstallUserspace` now is true by default, I think we don't need to specify it?

Reply:
Also, should we update the guidance of preparing containerized OVS in line 196-204?

Reply:
I would prefer to highlight "-InstallUserspace $true" with the context expecting to install Windows native Services.

Reply:
> I would prefer to highlight "-InstallUserspace $true" with the context expecting to install Windows native Services.

Got it, I am ok with it.

> Also, should we update the guidance of preparing containerized OVS in line 196-204?

@wenyingd Any follow-ups regarding this question?

Reply:
Updated, remove the original descriptions and example.